### ISSUE TITLE
Tokenizer Error: flake8 returns with code 0, which denotes success

### ISSUE THREAD

**Issue by asottile** [PyCQA/flake8#141]
In GitLab by @schtibe on Aug 11, 2016, 06:45

Best explained with an example:

```
$> echo "str(int(0)" > foo.py
$> flake8 foo.py # error happens here
$> echo $? # return code is 0
```

The error I get when executing flake8 is

```
Process Process-1:
Traceback (most recent call last):
  File "/usr/lib/python3.5/site-packages/flake8/processor.py", line 238, in generate_tokens
    for token in tokenize.generate_tokens(self.next_line):
  File "/usr/lib/python3.5/tokenize.py", line 597, in _tokenize
    raise TokenError("EOF in multi-line statement", (lnum, 0))
tokenize.TokenError: ('EOF in multi-line statement', (2, 0))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.5/site-packages/flake8/checker.py", line 46, in _run_checks_from_queue
    checker.run_checks(results_queue, statistics_queue)
  File "/usr/lib/python3.5/site-packages/flake8/checker.py", line 579, in run_checks
    self.process_tokens()
  File "/usr/lib/python3.5/site-packages/flake8/checker.py", line 554, in process_tokens
    for token in file_processor.generate_tokens():
  File "/usr/lib/python3.5/site-packages/flake8/processor.py", line 250, in generate_tokens
    raise exceptions.InvalidSyntax(exc.message, exception=exc)
AttributeError: 'TokenError' object has no attribute 'message'
```

In my opinion, the return code should be something else than 0

My Data:
* Python 3.5.2
* installed via pacman
* 3.0.3 (pyflakes: 1.2.3, pycodestyle: 2.0.0, mccabe: 0.5.2) CPython 3.5.2 on Linux

**Comment by asottile** [comment]
In GitLab by @sigmavirus24 on Nov 12, 2016, 11:44

Mentioned in merge request !142

**Comment by asottile** [comment]
In GitLab by @sigmavirus24 on Nov 12, 2016, 11:50

Status changed to closed by commit c81a403fefcde5545bd7d41c66efb7b658d166f5

**Comment by asottile** [comment]
In GitLab by @sigmavirus24 on Nov 12, 2016, 11:58

Mentioned in commit sigmavirus24/flake8@d53e6156e25f23e4266c92857175bba2a60ab183
